typegate/tests/e2e/website/website_test.ts

```diff
@@ -0,0 +1,101 @@
+// Copyright Metatype OÜ, licensed under the Elastic License 2.0.
+// SPDX-License-Identifier: Elastic-2.0
+
+import { expandGlob } from "std/fs/expand_glob.ts";
+import { dirname, fromFileUrl, join, resolve } from "std/path/mod.ts";
+import { existsSync } from "std/fs/exists.ts";
+import { copySync } from "std/fs/copy.ts";
+import { Meta } from "../../utils/mod.ts";
+import { MetaTest } from "../../utils/test.ts";
+import { assertEquals } from "std/assert/assert_equals.ts";
+import config from "../../../src/config.ts";
+
+export const thisDir = dirname(fromFileUrl(import.meta.url));
+
+function stripIncomparable(json: string) {
+  return [
+    // FIXME: python and deno does not produce the same tarball
+    (source: string) => source.replace(/"file:scripts\/.+;(.+)"/g, '""'),
+  ].reduce((prev, op) => op(prev), json);
+}
+
+async function testSerializeAllPairs(t: MetaTest, dirPath: string) {
+  const tempDir = Deno.makeTempDirSync({
+    dir: config.tmp_dir,
+  });
+
+  copySync(resolve(dirPath), tempDir, { overwrite: true });
+
+  const skip = [
+    // FIXME:
+    // "rest", // python and typescript generates different indentation for g.rest
```

Comment:
you can remove the spacing in the sdk

Reply:
Sorry that was a bad diagnosis, the graphql queries was already formatted on the sdk.

Seems like most of the issues I had where around the typegraph name, type `rename` (not equivalent to the old `{ name: <name>}`), the stacktrace was just a bit tough to debug for some typegraphs.

I did have formatting issues  with the python runtime though, the python frontend parse its own source then makes some transformation to the function/lambda provided so it's never stricly equal to the typescript version unless we see what python generate first then give it to the typescript version.